mapentity/settings.py

```diff
@@ -44,6 +44,7 @@
     'MAPENTITY_WEASYPRINT': False,
     'MAP_STYLES': _DEFAULT_MAP_STYLES,
     'REGEX_PATH_ATTACHMENTS': r'\.\d+x\d+_q\d+(_crop)?\.(jpg|png|jpeg)$',
+    'MAX_CHARACTERS': 0,
```

Comment:
```suggestion
    'MAX_CHARACTERS': None,
```
It's better to use None instead of value which is confusing